Description: debianization -- goto-cc analysis workaround
 Meant to (hopefully) neutralize bugs #748142.
Origin: debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2014-09-16

--- a/macro.h
+++ b/macro.h
@@ -816,10 +816,6 @@
 #define FATALERROR(text) fatal_error(text)
 #define CH2D(a,b) if ((a) == (b) ) { erg += equal_2_error(); goto endr_ende;}
 
-/*
-#define CE2(a,b,f) if (check_equal_2(a,b,f,&erg) == EQUAL) goto endr_ende;
-*/
-
 #define CE2A(a,b,f) if (check_equal_2a(a,b,f,&erg) == EQUAL) goto endr_ende;
 
 #define CE3(a,b,c,f) \
